content/about/funding.md

```diff
@@ -3,18 +3,18 @@ $title@: Funding
 $order: 7
 ---
 
-This page lists currently active funding for our projects
+<img style="float: right;" src="https://bit.ly/2XPyew9" width=200 height=200>
 
-#### Synaptomes of Mouse and Man
+#### [Synaptomes of Mouse and Man](http://grantome.com/grant/NIH/R01-NS092474-01)
```

Comment:
this grant is now over and should remove?

Reply:
good catch, will do